submissions/Ant-C-tech/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,466 @@
+            <p>From<br>01 March<br>1h 39m</p>
+        </div>
+
+        <div class="mask"></div>
+
+        <img
+          class="posterSection__img"
+          src="img/poster.jpg"
+          alt="vivarium poster"
+        />
+      </section>
+
+      <section class="controlSection">
+        <nav class="movieType">
+          <ul class="movieType__list">
+            <li class="movieType__listItem movieType__listItem-disabled">
+              <a class="movieType__listItemLink movieType__listItemLink-disabled" href="#">General</a>
+            </li>
+            <li class="movieType__listItem">
+              <a class="movieType__listItemLink" href="#">Digital 2D</a>
+            </li>
+            <li class="movieType__listItem movieType__listItem-active">
+              <a class="movieType__listItemLink movieType__listItemLink-active" href="#">IMAX 3D</a>
+            </li>
+            <li class="movieType__listItem">
+              <a class="movieType__listItemLink" href="#">Cinema 4Dx</a>
+            </li>
+          </ul>
+        </nav>
+
+        <nav class="movieDate">
+          <p class="movieDate__title">Tue, 2 March</p>
+          <div class="flexRow">
+            <a href="#" class="btn-prevDay">&#9668;</a>
+              <ol class="calendar">
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink calendar__listItemLink">
+                    <span class="calendar__supText calendar__supText-today">Today</span><br>1
+                  </a>
+                </li>
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink calendar__listItemLink-active">
+                    <span class="calendar__supText calendar__supText-active">Tue</span><br>2
+                  </a>
+                </li>
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink">
+                    <span class="calendar__supText">Wed</span><br>3
+                  </a>
+                </li>
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink">
+                    <span class="calendar__supText">Thu</span><br>4
+                  </a>
+                </li>
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink calendar__listItemLink-disabled">
+                    <span class="calendar__supText calendar__supText-disabled">Fri</span><br>5
+                  </a>
+                </li>
+              </ol>
+            <a href="#" class="btn-nextDay">	&#9658;</a>
+          </div>
+        </nav>
+
+        <nav class="movieTime">
+          <p class="movieTime__title">Time</p>
+          <ul class="movieTime__list">
+            <li class="movieTime__listItem  movieTime__listItem-active">
+              <a class="movieTime__listItemLink  movieTime__listItemLink-active" href="#">14:00</a>
```

Comment:
For date and time preferable to use `<time>` tag [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)